guides/common/modules/ref_best-practices-for-patching-content-hosts.adoc

```diff
@@ -1,7 +1,7 @@
 [id="best-practices-for-patching-content-hosts_{context}"]
 = Best practices for patching content hosts
 
-* Registering hosts to {Project} requires {project-client-name}, which contains the `subscription-manager` package, `katello-host-tools` package, and their dependencies.
+* Registering hosts to {Project} requires {project-client-name}, which contains the `katello-host-tools-tracer` package and its dependencies.
```

Comment:
```suggestion
* Registering hosts to {Project} requires {project-client-name}, which contains the `katello-host-tools` package and its dependencies.
```
It contains both. I'm not sure about changing the package name, but it isn't a blocker.

Reply:
Fixed.